src/ex4/server/clients/pokemon_client.js

```diff
@@ -1 +1,21 @@
-// The Pokemon Client (using axios) goes here
\ No newline at end of file
+// The Pokemon Client (using axios) goes here
+import dotenv from "dotenv";
+import axios from "axios";
+dotenv.config();
+const envFile = process.env;
```

Comment:
a line of space after 5 will make the code tidier, also why not use envModule here if you already have it?